doc: audio: app-specific kconfigs (#22942)
nrfconnect/sdk-nrf: applications/nrf5340_audio/doc/configuration_options.rst

.. _config_audio_app_options:

nRF5340 Audio: Application-specific Kconfig options
###################################################

.. contents::
   :local:
   :depth: 2

> Following are the application-specific configuration options that can be configured for the nRF5340 Audio applications and their internal modules:

Reviewer: ```suggestion
You can configure the following application-specific configuration options for the nRF5340 Audio applications and their internal modules:
```

Author: This is identical to docs for nRF Desktop and nRF Machine Learning apps, so I won't make this change here. If you want it changed, please open a JIRA so it can be aligned there too.
